apps/docs/src/app/(main)/dashboard/page.tsx

```diff
@@ -0,0 +1,163 @@
+'use client'
+
+import { useEffect, useState } from 'react'
+import { BarChart, CreditCard, Key, LoaderIcon, LucideIcon, Settings, Users } from 'lucide-react'
+
+import { BillingContent } from '@/components/Dashboard/Billing'
+import { TeamContent } from '@/components/Dashboard/Team'
+
+import { Team, useUser } from '@/utils/useUser'
+import { User } from '@supabase/supabase-js'
+import { KeysContent } from '@/components/Dashboard/Keys'
+import { UsageContent } from '@/components/Dashboard/Usage'
+import { AccountSelector } from '@/components/Dashboard/AccountSelector'
+import { useRouter, useSearchParams } from 'next/navigation'
+import { PersonalContent } from '@/components/Dashboard/Personal'
+import Link from 'next/link'
+import { Button } from '@/components/Button'
+
+function redirectToCurrentURL() {
+  const url = typeof window !== 'undefined' ? window.location.href : undefined
+
+  if (!url) {
+    return ''
+  }
+
+  const encodedURL = encodeURIComponent(url)
+  return `redirect_to=${encodedURL}`
+}
+
+// TODO: Sandbox and Templates tab deleted for now
+const menuLabels = ['personal', 'keys', 'usage' ,'billing', 'team',] as const
+type MenuLabel  = typeof menuLabels[number]
+
+export default function Dashboard() {
+  const { user, isLoading, error } = useUser()
+  const router = useRouter()
+  const searchParams = useSearchParams()
+
+  const tab = searchParams!.get('tab')
```

Comment:
you could also add team_id, if I have multiple teams and I do refresh, it will choose the default one